docs/publish-docker-images.md

```diff
@@ -21,14 +21,18 @@ You can use GitHub actions to publish Docker images to the Docker Hub repository
    + Selecting the main branch results in publishing the Docker image with the specified tag and the latest tag.
    + Selecting another branch, results are publishing the Docker image with the specified tag.
 4. Choose the image you want to build
-   + `datastrato/gravitino-ci-hive`.
-   + `datastrato/gravitino-ci-trino`.
+   + `datastrato/gravitino-ci:hive`.
+   + `datastrato/gravitino-ci:trino`.
    + Future plans include support for other data sources.
-5. Input the `tag name`, for example: `0.1.0`, Then build and push the Docker image name as `datastrato/{image-name}:0.1.0`.
-6. You must enter the correct `publish docker token` before you can execute run `Publish Docker Image` workflow.
-7. Wait for the workflow to complete. You can see a new Docker image shown in the [Datastrato Docker Hub](https://hub.docker.com/u/datastrato) repository.
-
-![Publish Docker image](assets/publish-docker-image.png)
+5. Input the `tag name`, for example: `0.1.0`, Then build and push the Docker image name. Currently, the Docker image name is in the format:
+   1. `datastrato/gravitino-ci:{image-type}-0.1.0` if this is a trino CI image, image-type is `trino`, `hive`, `kerberos-hive`, `doris`, `ranger`.
+   2. `datastrato/gravitino-playground:{image-type}-0.1.0` if this is a playground image, image-type is `trino`, `hive`, `ranger`.
+   3. `datastrato/gravitino:0.1.0` if this is a gravitino server image.
+   4. `datastrato/gravitino-iceberg-rest:0.1.0` if this is an iceberg-rest server image.
```

Comment:
Why do we still use datastrato here?

Reply:
I have completed checked all place that user docker hub username `datastrao`.